Geometry/TrackerNumberingBuilder/plugins/TrackerGeometricDetESModule.cc

```diff
@@ -24,9 +24,30 @@ using namespace edm;
 
 TrackerGeometricDetESModule::TrackerGeometricDetESModule( const edm::ParameterSet & p ) 
   : fromDDD_( p.getParameter<bool>( "fromDDD" )),
-    layerNumberPXB_( p.exists( "layerNumberPXB" ) ? p.getParameter<unsigned int>( "layerNumberPXB" ) : 16U ),// 16 for current, 18 for SLHC 
-    totalBlade_( p.exists( "totalBlade" ) ? p.getParameter<unsigned int>( "totalBlade" ) : 24U )             // 24 for current, 56 for SLHC 
```

Comment:
Removing these two lines, plus fillDescriptions checks, now break HLT:
```
----- Begin Fatal Exception 06-Jan-2015 07:29:23 CET-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Validating configuration of ESProducer or ESSource of type TrackerGeome\
tricDetESModule with label: 'TrackerGeometricDetESModule'
Exception Message:
Illegal parameters found in configuration.  The parameters are named:
 'layerNumberPXB'
 'totalBlade'
You could be trying to use parameter names that are not
allowed for this plugin or they could be misspelled.
----- End Fatal Exception -------------------------------------------------
```

I wonder why this was not flagged by jenkins tests.